docs/pages/guides/in-app-purchases.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Use In-app purchase
+description: Learn about how to use in-app purchases in your Expo app.
+hideTOC: true
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { CODE } from '~/ui/components/Text';
+import { BookOpen02Icon, GithubIcon } from '@expo/styleguide-icons';
+
+In-app purchases (IAP) are transactions within a mobile or desktop application where users can buy digital goods or additional features. This guide provides a list of popular libraries and tutorials for implementing IAP in your Expo app.
+
+> In-app purchase libraries require configuring custom native code. Native code is not configurable when using Expo Go. Instead, create a [development build](/develop/development-builds/introduction/), which allows using a native library in your project.
+
+## Tutorial
+
+<BoxLink
+  title="Expo In-App Purchase Tutorial"
+  description={
+    <>
+      The getting started guide for in-app purchases and subscriptions with{' '}
+      <CODE>react-native-purchases</CODE> library and RevenueCat.
+    </>
+  }
+  href="https://www.revenuecat.com/blog/engineering/expo-in-app-purchase-tutorial/"
+  Icon={BookOpen02Icon}
+/>
+
+## Libraries
+
```

Comment:
```suggestion

These libraries provide robust support for in-app purchase functionality, as well as out-of-the-box compatibility with Expo apps using [CNG](workflow/continuous-native-generation/). 
```
Some kind of intro as to why we're recommending these (they have config plugins + Expo-specific docs)